src/components/Window.js

```diff
@@ -41,9 +41,9 @@ export class Window extends Component {
       <div id={window.id} className={ns('window')}>
         {this.wrappedTopBar()}
         <WindowMiddleContent
-          window={window}
           manifest={manifest}
           sideBarOpen={window.sideBarOpen}
+          window={window}
```

Comment:
If the intention is the sort the props by name then it's off topic. Same applies to WindowMiddleContent and WindowViewer.